db/init_db.js

```diff
@@ -25,7 +73,9 @@ async function populateInitialData() {
   }
 }
```

Comment:
Call your db adapter functions in `populateInitialData` so that it is all tested!

Something like
```js
createUser({firstName: 'Preston', lastName: 'Wallace'}); // ... etc
```
Just basically call each function with something valid, and `console.log()` the result (in this seed file) to confirm 
1. doesn't error
2. creates (or queries) the correct data